<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passport_widget</class>
 <widget class="QWidget" name="passport_widget">
  <property name="geometry">
   <rect>
	<x>0</x>
	<y>0</y>
	<width>435</width>
	<height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Паспорт</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
	<number>0</number>
   </property>
   <property name="topMargin">
	<number>10</number>
   </property>
   <property name="rightMargin">
	<number>10</number>
   </property>
   <property name="bottomMargin">
	<number>10</number>
   </property>
   <item>
	<widget class="QGroupBox" name="passport_groupBox">
	 <property name="font">
	  <font>
	   <pointsize>16</pointsize>
	   <weight>75</weight>
	   <bold>true</bold>
	  </font>
	 </property>
	 <property name="title">
	  <string>Паспортные данные</string>
	 </property>
	 <layout class="QGridLayout" name="gridLayout">
	  <property name="leftMargin">
	   <number>9</number>
	  </property>
	  <property name="topMargin">
	   <number>9</number>
	  </property>
	  <property name="horizontalSpacing">
	   <number>6</number>
	  </property>
	  <item row="1" column="0">
	   <widget class="QLabel" name="surname_label">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <pointsize>16</pointsize>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="text">
		 <string>Фамилия</string>
		</property>
	   </widget>
	  </item>
	  <item row="0" column="0">
	   <widget class="QLabel" name="name_label">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <pointsize>16</pointsize>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="text">
		 <string>Имя</string>
		</property>
	   </widget>
	  </item>
	  <item row="5" column="1" colspan="2">
	   <widget class="QLineEdit" name="issue_lineEdit">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="styleSheet">
		 <string notr="true"> QLineEdit {
	  border-width: 1px;
	  border-style: solid;
	  border-radius: 0px;
  }</string>
		</property>
	   </widget>
	  </item>
	  <item row="3" column="2">
	   <widget class="QRadioButton" name="female_radioButton">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="text">
		 <string>Ж</string>
		</property>
	   </widget>
	  </item>
	  <item row="1" column="1" colspan="2">
	   <widget class="QLineEdit" name="surname_lineEdit">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <pointsize>16</pointsize>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="styleSheet">
		 <string notr="true"> QLineEdit {
	  border-width: 1px;
	  border-style: solid;
	  border-radius: 0px;
  }</string>
		</property>
	   </widget>
	  </item>
	  <item row="2" column="1" colspan="2">
	   <widget class="QLineEdit" name="patr_lineEdit">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <pointsize>16</pointsize>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="styleSheet">
		 <string notr="true"> QLineEdit {
	  border-width: 1px;
	  border-style: solid;
	  border-radius: 0px;
  }</string>
		</property>
	   </widget>
	  </item>
	  <item row="4" column="1" colspan="2">
	   <widget class="QDateEdit" name="birth_dateEdit">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <pointsize>16</pointsize>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="styleSheet">
		 <string notr="true"> QDateEdit {
	  border-width: 1px;
	  border-style: solid;
	  border-radius: 0px;
  }</string>
		</property>
	   </widget>
	  </item>
	  <item row="5" column="0">
	   <widget class="QLabel" name="issue_label">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="text">
		 <string>Дата выдачи</string>
		</property>
	   </widget>
	  </item>
	  <item row="3" column="0">
	   <widget class="QLabel" name="sex_label">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="text">
		 <string>Пол</string>
		</property>
	   </widget>
	  </item>
	  <item row="2" column="0">
	   <widget class="QLabel" name="patr_label">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <pointsize>16</pointsize>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="text">
		 <string>Отчество</string>
		</property>
	   </widget>
	  </item>
	  <item row="3" column="1">
	   <widget class="QRadioButton" name="male_radioButton">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="text">
		 <string>М</string>
		</property>
	   </widget>
	  </item>
	  <item row="0" column="1" colspan="2">
	   <widget class="QLineEdit" name="name_lineEdit">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <pointsize>16</pointsize>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="styleSheet">
		 <string notr="true"> QLineEdit {
	  border-width: 1px;
	  border-style: solid;
	  border-radius: 0px;
  }</string>
		</property>
	   </widget>
	  </item>
	  <item row="4" column="0">
	   <widget class="QLabel" name="birth_label">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="text">
		 <string>Дата рождения</string>
		</property>
	   </widget>
	  </item>
	  <item row="7" column="1" colspan="2">
	   <widget class="QLineEdit" name="num_lineEdit">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="styleSheet">
		 <string notr="true"> QLineEdit {
	  border-width: 1px;
	  border-style: solid;
	  border-radius: 0px;
  }</string>
		</property>
	   </widget>
	  </item>
	  <item row="8" column="1" colspan="2">
	   <widget class="QLineEdit" name="nationality_lineEdit">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="styleSheet">
		 <string notr="true"> QLineEdit {
	  border-width: 1px;
	  border-style: solid;
	  border-radius: 0px;
  }</string>
		</property>
		<property name="inputMask">
		 <string>9999 999999</string>
		</property>
	   </widget>
	  </item>
	  <item row="8" column="0">
	   <widget class="QLabel" name="nationality_label">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="text">
		 <string>Национальность</string>
		</property>
	   </widget>
	  </item>
	  <item row="7" column="0">
	   <widget class="QLabel" name="num_label">
		<property name="minimumSize">
		 <size>
		  <width>0</width>
		  <height>30</height>
		 </size>
		</property>
		<property name="font">
		 <font>
		  <weight>50</weight>
		  <bold>false</bold>
		 </font>
		</property>
		<property name="text">
		 <string>Серия, номер</string>
		</property>
	   </widget>
	  </item>
	 </layout>
	</widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name_lineEdit</tabstop>
  <tabstop>surname_lineEdit</tabstop>
  <tabstop>patr_lineEdit</tabstop>
  <tabstop>male_radioButton</tabstop>
  <tabstop>female_radioButton</tabstop>
  <tabstop>birth_dateEdit</tabstop>
  <tabstop>issue_lineEdit</tabstop>
  <tabstop>num_lineEdit</tabstop>
  <tabstop>nationality_lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
